pkg/js/util/parser-typings.ts

```diff
@@ -0,0 +1,48 @@
+
+
+export enum RewriteType {
```

Comment:
You don't need any of the types in this file, these are for the syntax transformer intermediary json - you don't have that here